spec/rspec/rails/matchers/have_rendered_spec.rb

```diff
@@ -1,8 +1,10 @@
 %w[have_rendered render_template].each do |template_expectation|
-  describe template_expectation do
+  RSpec.describe template_expectation do
     include RSpec::Rails::Matchers::RenderTemplate
     let(:response) { ActionDispatch::TestResponse.new }
 
+    def assert_template(*); end # Stub method
```

Comment:
Why did you move this?

Reply:
Not only that example that I've moved it from need `assert_template` to be defined on `self`.
I was on the fence of keeping it there for clarity. Should I?

Reply:
It makes it available to more specs, which is not necessarily desired.